src/main/java/io/cryostat/messaging/MessagingServer.java

```diff
@@ -200,14 +197,15 @@ public void writeMessage(WsMessage message) {
     }
 
     @Override
-    public void close() {
-        if (prunerTask != null) {
-            prunerTask.cancel(false);
-        }
+    public void stop() {
+        this.close();
+    }
 
-        if (pingTask != null) {
-            pingTask.cancel(false);
-        }
+    @Override
+    public void close() {
+        this.vertx.cancelTimer(prunerTaskId);
+        pingTaskIds.forEach(vertx::cancelTimer);
+        pingTaskIds.clear();
```

Comment:
If this `close()` method removes all the `pingTaskIds` at once, does that mean `removeConnection()` for a single connection should remove a single `pingTaskId`?